pylint/reporters/base_reporter.py

```diff
@@ -23,6 +23,9 @@ class BaseReporter:
 
     extension = ""
 
+    name = "base"
```

Comment:
```suggestion
    name: str = "base"
```
Or is that implicit because we assign a string to it ?

Reply:
Yes, it is! 😄